docs/data-ingestion/Proxmox.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Proxmox
+weight: 1
+sort: 1
+menu:
+  docs:
+    identifier: "proxmox"
+    parent: "data-ingestion"
+    weight: 1
+    # sort: 1
+aliases:
+  - /data-ingestion/proxmox.html
+  - /data-ingestion/Proxmox.html
+---
+
+# Proxmox Data Ingestion
+Since Proxmox Virtual Environment(PVE) and Proxmox Backup Server(PBS) support sending data using the InfluxDB We can use the InfluxDB write support built into VictoriaMetrics
+Currently PVE and PBS only support using an Authorization Token for authentication and does not support basic auth or a username and password.
+
+## Proxmox Virtual Environment (PVE)
+If want help Sending your data to Managed VictoriaMetrics check out [our blog](https://victoriametrics.com/blog/proxmox-monitoring-with-dbaas/).
+
+1. Login to PVE as an administrator
+2. Go to DataCenter > MetricServer > Add > InfluxDB
+3. Set the parameters as follows
+  * Name: VictoriaMetrics (can be changed to any string)
+  * Server: the hostname or IP of your VictoriaMetrics Instance
+  * Port: This will vary depending on where you were you sending the data and your settings but here are the defaults for the various components
+    * 8428 for sending directly to VMInsert or VMSingle
+    * 8427 for VMAuth
+    * 8429 for VMAgent
+  * Protocol: use HTTPS if you have TLS/SSL configured otherwise use HTTPS
+  * Organization: leave empty since it doesn't get used
```

Comment:
"Protocol: use HTTPS if you have TLS/SSL configured otherwise use `HTTPS`" -> Should be `HTTP`